test/.eslintrc.json

```diff
@@ -8,6 +8,7 @@
 		"sinon": true
 	},
 	"rules": {
-		"arrow-body-style": "off"
+		"arrow-body-style": "off",
+		"no-unused-expressions": "off"
```

Comment:
:'(

Reply:
This is because when we use `chai/should`, we have property call everywhere like
`defaultPublicKeyCurve.equals(Buffer.from(curveRepresentation)).should.be.true;`

Reply:
Yeah, I just don't like this aspect of Chai.